src/realm/exec/realm2json.cpp

```diff
@@ -2,35 +2,73 @@
 #include <iostream>
 #include <realm/history.hpp>
 
+const char* legend = "Simple tool to output the JSON representation of a Realm:\n"
+                     "  realm2json [-link-depth=N] [-output-mode=N] <.realm file>\n"
+                     "\n"
+                     "Options:\n"
+                     " -link-depth: How deep to traverse linking objects (use -1 for infinite). See test_json.cpp "
+                     "for more details. Defaults to 0.\n"
+                     " -output-mode:Can specify MongoDB XJSON output format by passing in 1. Defaults to 0. \n"
```

Comment:
seems like this flag would be better off as a `--xjson` flag which just defaults to "off" when excluded.

Reply:
Sounds good, was trying to keep it in line with the fact that output_mode is just an int, but I agree that we will most likely not have another output type and can cross that bridge if we get there